src/Atmos/Model/atmos_tendencies.jl

```diff
@@ -13,51 +13,184 @@ eq_tends(pv::AbstractPrognosticVariable, m::AtmosModel, tt::Source) =
 ##### First order fluxes
 #####
 
-eq_tends(::Mass, ::Anelastic1D, ::Flux{FirstOrder}) = ()
+# Dispatch to flux differencing form
+eq_tends(
+    pv::AbstractPrognosticVariable,
+    atmos::AtmosModel,
+    tt::Flux{FirstOrder},
+) = eq_tends(atmos.equations_form, pv, atmos, tt)
+eq_tends(
+    pv::AbstractPrognosticVariable,
+    atmos::AtmosModel,
+    tt::FluxDifferencing{FirstOrder},
+) = eq_tends(atmos.equations_form, pv, atmos, tt)
+
+# Default is no flux differencing
 
-eq_tends(::Mass, ::Compressible, ::Flux{FirstOrder}) = (Advect(),)
+eq_tends(
+    ::AbstractEquationsForm,
+    ::AbstractPrognosticVariable,
+    ::AtmosModel,
+    ::FluxDifferencing{FirstOrder},
+) = ()
+
+# helpers
 
 # Mass
-eq_tends(pv::Mass, atmos::AtmosModel, tt::Flux{FirstOrder}) =
-    (eq_tends(pv, compressibility_model(atmos), tt))
+eq_tends(
+    ::Mass,
+    ::Anelastic1D,
+    ::Union{Flux{FirstOrder}, FluxDifferencing{FirstOrder}},
```

Comment:
```suggestion
    ::AbstractFluxTendency{FirstOrder},
```
might help with the formatting (and elsewhere)